rosidl_generator_cpp/resource/msg__traits.hpp.em

```diff
@@ -29,10 +29,6 @@ cpp_namespace = '%s::%s::' % (spec.base_type.pkg_name, subfolder)
 }@
 #include <stdint.h>
 
-#include <array>
-#include <string>
-#include <vector>
```

Comment:
You might need to add `#include <type_traits>` here, since `std::integral_constant` is used below, see:

http://en.cppreference.com/w/cpp/types/integral_constant

I don't know if maybe one of these was previously bringing that symbol in for you.